eg5.py

```diff
@@ -0,0 +1,35 @@
+"""You are given a list of numbers and your task is to find out the pairss that have sum equals n (provided during input)."""
+
+
+numbers = [9, 4, 8, 10, 2, 4, 8, 3, 14, 4, 8]    #Given List
+n=12        #input for check sum of two pair
+pairs=[]    #it will store pair whose sum is n
+
+for number in numbers:
+    for item in range(len(numbers)):
+        if(number+numbers[item]==n):
+            pairs.append([number,numbers[item]])  #it store the two number whose sum is n as list
```

Comment:
I think this will duplicate the pairs as well. For eg, it will add the [4, 8] and [8, 4] as well even though both of the pair are same.